src/Core/Settings.h

```diff
@@ -280,6 +280,7 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     M(UInt64, http_max_fields, 1000000, "Maximum number of fields in HTTP header", 0) \
     M(UInt64, http_max_field_name_size, 1048576, "Maximum length of field name in HTTP header", 0) \
     M(UInt64, http_max_field_value_size, 1048576, "Maximum length of field value in HTTP header", 0) \
+    M(UInt64, http_max_chunk_size, 32*1024*1024, "Maximum value of a chunk size in HTTP chunked transfer encoding", 0) \
```

Comment:
imo this value is too restrictive. we want just to protect ourselves from overflow. so maybe pick something that "ought to be enough for everyone", like 100GB or SLT?